drivers/auth/auth_mod.c

```diff
@@ -59,9 +58,9 @@ extern unsigned int auth_img_flags[];
 static int cmp_auth_param_type_desc(const auth_param_type_desc_t *a,
 		const auth_param_type_desc_t *b)
 {
-	if ((a->type == b->type) && (a->cookie == b->cookie)) {
+	if (a->type == b->type && a->cookie == b->cookie)
```

Comment:
Although the parenthesis are not needed in this case, having them there makes the code easier to read and more critically there are static analysis rules that TF is looking to support that state that code should not rely on operator precedence (and so use parenthesis to enforce).